crypto/x509v3/v3_asid.c

```diff
@@ -830,6 +859,10 @@ static int asid_validate_path_internal(X509_STORE_CTX *ctx,
  */
 int X509v3_asid_validate_path(X509_STORE_CTX *ctx)
 {
+    if (ctx->chain == NULL
```

Comment:
But if this check would be considered legitimate, would it be appropriate to set ctx->error?